apps/dot_ext/oauth2_validators.py

```diff
@@ -1,9 +1,33 @@
-from oauth2_provider.oauth2_validators import OAuth2Validator
+from oauth2_provider.oauth2_validators import OAuth2Validator as DotOAuth2Validator
 from django.core.exceptions import ObjectDoesNotExist
 from apps.pkce.oauth2_validators import PKCEValidatorMixin
 from oauthlib.oauth2.rfc6749.errors import InvalidGrantError
 
 
+class OAuth2Validator(DotOAuth2Validator):
+    def _extract_basic_auth(self, request):
+        """
+        This overrided method additionally checks the "Authorization" header
+        for compatability.
+        TODO: Remove this if it becomes possible.
+        """
+        auth = request.headers.get("HTTP_AUTHORIZATION", None)
+        if not auth:
+            auth = request.headers.get("Authorization", None)
```

Comment:
This override is to support a change in behavior where the request headers are being set and the authorization is now contained in the `Authorization` header.